TASVideos/Pages/Forum/Subforum/Index.cshtml

```diff
@@ -38,8 +38,15 @@
 					</a>
 				</td>
 				<td>@Math.Max(0, topic.PostCount - 1)</td> @*Imported Submission Topics will have no replies and no original post*@
-				<td>@topic.CreateUserName</td>
-				<td><timezone-convert asp-for="@topic.LastPostDateTime" /></td>
+				<td> <profile-link username="@topic.CreateUserName">@topic.CreateUserName</profile-link></td>
+                <td>
+                    @if (topic.LastPost != null)
+                    {
+                        <timezone-convert asp-for="@topic.LastPostDateTime" /> <br />
+                        <profile-link username="@topic.LastPostUserName">@topic.LastPostUserName</profile-link>
```

Comment:
This taghelper allows you to omit the value between the tags.  so you can simply do:
<profile-link username="@topic.LastPostUserName"></profile-link>